docs/home.md

```diff
@@ -57,93 +84,135 @@ underlying [Marinade Finance protocol](https://docs.marinade.finance/faq/faq#wha
-their operations.
+The yield acrrued on the staked SOL, is equal to the value of mSOL and bSOL held by Sunrise minus the value of SOL staked (calculated as the circulatinbg supply of gSOL).
 
-Once the pool is established, Sunrise will direct stake to this pool, resulting in positive
-climate impact on both sides of the transaction:
+###### Liquidity Pool
 
-- Validators have an incentive to use renewable energy, reducing the carbon cost of the Solana network.
-- Stake yield is directed towards offsetting carbon emissions outside the solana network.
 
-### Where does the earned yield go?
+### Unstaking
+If you unstake your SOL, Sunrise calculates how much mSOL to sell back to Marinade to receive your SOL
+For example, if you decides to unstake 1 SOL, Sunrise will calculate how much mSOL to sell to Marinade (and bSOL to sell back to BlazeStake), in order to receive back that 1 SOL.
+* Once the latest update is made to send staked SOL to the Marinade Liquidity Pool, Sunrise will be selling back a mix of mSOL and Liquidity Pool tokens to Marinade.
 
-While the protocol is in development, yield remains in the protocol itself. 
+                                                                    
+## Sunrise Stake treasury spending 
 
-In the future, yield will be directed to purchasing carbon offset tokens, which will be
-subsequently burned in order to retire the underlying carbon credits.
+Thanks to the yield earned in the staking process, Sunrise can purchase carbon tokens and burn them to offset carbon emissions. 
 
-Long-term, the decision of where to direct yield will be made by the community, governed by
-the [Sunrise Stake DAO](https://app.realms.today/realm/SunriseStake).
+#### 1. Yield transfer to the treasury account
 
-## Governance
+The yield earned from staking SOL is transferred to the __treasury account__. It is a Program Derived Address ([PDA](https://docs.solana.com/developing/programming-model/calling-between-programs#program-derived-addresses)) meaning that it is owned and controlled by a smart contract, not a private key. This smart contract is known as the __treasury controller__. It can trigger a state change to buy or burn carbon tokens, thus retiring underlying carbon credits.
 
-:::tip
+#### 2. Purchasing carbon tokens
+The carbon token used in Sunrise Stake is the Nature Carbon Tonne token. NCT represents a tonne of CO2 or equivalent greenhouse gas removed from the atmosphere in various nature-based projects. It is issued on the Polygon and Celo blockchains. More on the NCT can be found on Toucan’s [blog](https://blog.toucan.earth/announcing-nct-nature-carbon-tonne/) or [GitHub](https://github.com/ToucanProtocol/contracts) repository.
 
-Governance for Sunrise Stake is planned for 2023. Stay tuned!
+As the controller cannot purchase tokens directly from a different chain, Sunrise Stake employs a __holding account__ to buy NCT by bridging it to Solana through [Wormhole](https://www.portalbridge.com/#/transfer).
```

Comment:
No need for this line _and_ 158. I think 158 is sufficient. Maybe add the wormhole detail there.